frontend/src/auth/components/EmailVerificationPage/styles.scss

```diff
@@ -0,0 +1,19 @@
+@import '~styles/theme';
+
+
+.email-verification-container {
+    justify-content: center;
+    text-align: center;
+}
+
+i.email-verification-title-icon {
+    padding-left: 15px;
+}
+
+.email-verification-console-button {
+    margin-bottom: 20px;
+
+    i {
+        padding-left: 10px;
+    }
+}
```

Comment:
@giovannicimolin, missing newline at the end of the file.

Reply:
Done!